.github/workflows/docker.yml

```diff
@@ -88,4 +73,13 @@ jobs:
           echo "Tagging armory-docker-all.jfrog.io/armory/$BASE_PROJECT$RELEASE to armory/$BASE_PROJECT:$RELEASE"
           docker image tag armory-docker-local.jfrog.io/armory/$BASE_PROJECT:$RELEASE armory/$BASE_PROJECT:$RELEASE
           echo "Pushing image..."
-          docker push armory/$BASE_PROJECT:$RELEASE
\ No newline at end of file
+          docker push armory/$BASE_PROJECT:$RELEASE
+
+      - name: Run Security Scan
```

Comment:
Why do this after publishing to docker hub? I think we'd want the scan results before proceeding no?

Reply:
Moved!